src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/CommunityOffenderManager.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity
+
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.ManyToMany
+import javax.persistence.OneToMany
+import javax.persistence.Table
+import javax.validation.constraints.NotNull
+
+@Entity
+@Table(name = "community_offender_manager")
+data class CommunityOffenderManager(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @NotNull
+  val id: Long = -1,
+
+  @Column(unique = true)
+  val staffIdentifier: Long,
+
+  @Column(unique = true)
+  val username: String,
+
```

Comment:
Do we need their `staffCode` in here too? Anything need to use it?

Reply:
We dont use staffCode for anything